### PR TITLE
420 add feedback

### PR DESCRIPTION
**Pull request by rendisaepulb**
#420 Feedback

### Breakdown:
- [x] add feedback
- [x] change localization string

### CODE REVIEW

File: app/containers/Settings/index.js [hunk 1]
```diff
@@ -21,14 +22,18 @@ import * as actions from './actions';
...
+  constructor(props) {
+    super(props);
+    this.setModalVisible = this.setModalVisible.bind(this);
+    this.state = {
+      id: null,
+      modalVisible: false,
+      firstName: '',
+      lastName: '',
+      photo: null,
+      scalesPageToFit: true
```
Comment (andy-shi88) [inline app/containers/Settings/index.js]: we never change this value right? if so I don't think we need to add state for this

File: app/containers/Settings/index.js [hunk 2]
```diff
@@ -77,6 +82,8 @@ class Settings extends Component {
       summary,
       profilePic
     } = fields || '';
+    const WEBVIEW_REF = 'webview';
+    const feedback = 'https://devsummit.io/feedback.html'
```
Comment (andy-shi88) [inline app/containers/Settings/index.js]: can you set the feedback_url in the global constants instead?

File: app/localization/id/Settings.js [hunk 3]
```diff
@@ -3,5 +3,6 @@ export default {
   editProfile: 'Edit profil',
   codeConduct: 'Kode etik',
   privacyPolicy: 'Kebijakan pribadi',
-  logout: 'Keluar'
+  logout: 'Keluar',
+  feedback: 'Umpan Balik'
```
Comment (andy-shi88) [inline app/localization/id/Settings.js]: change it to `kritik dan saran`

File: app/containers/Settings/index.js [hunk 4]
```diff
@@ -121,6 +135,30 @@ class Settings extends Component {
...
+                <TouchableOpacity onPress={() => this.setModalVisible(!this.state.modalVisible)}>
+                  <Icon name={'ios-arrow-dropleft'} size={24} color={'black'} style={{ padding: 10 }} />
+                </TouchableOpacity>
+              </View>
+              <WebView
+                automaticallyAdjustContentInsets={false}
+                source={{ uri: feedback }}
+                style={{ marginTop: 20 }}
+                scalesPageToFit={this.state.scalesPageToFit}
+                ref={WEBVIEW_REF}
```
Comment (andy-shi88) [inline app/containers/Settings/index.js]: the value will always be `webview` right? then we need no variable for this, just add it here as plain string.

**Review by andy-shi88 — CHANGES_REQUESTED** [review]

File: app/containers/Settings/index.js (hunk 1 above)
Comment (rendisaepulb) [reply app/containers/Settings/index.js]: ok

File: app/containers/Settings/index.js (hunk 2 above)
Comment (rendisaepulb) [reply app/containers/Settings/index.js]: ok

File: app/containers/Settings/index.js (hunk 4 above)
Comment (rendisaepulb) [reply app/containers/Settings/index.js]: i see

File: app/localization/id/Settings.js (hunk 3 above)
Comment (rendisaepulb) [reply app/localization/id/Settings.js]: ok

File: app/containers/Settings/index.js (hunk 1 above)
Comment (rendisaepulb) [reply app/containers/Settings/index.js]: so we can remove the state ?

Comment (andy-shi88) [reply app/containers/Settings/index.js]: yes.

File: app/containers/Settings/constants.js [hunk 5]
```diff
@@ -4,3 +4,4 @@ export const UPDATE_AVATAR = 'app/containers/Profile/UPDATE_AVATAR';
 export const UPDATE_IS_AVATAR_UPDATED = 'app/containers/Profile/UPDATE_IS_AVATAR_UPDATED';
 export const UPDATE_IS_LOG_OUT = 'app/containers/Profile/UPDATE_IS_LOG_OUT';
 export const UPDATE_IS_DISABLED = 'app/containers/Profile/UPDATE_IS_DISABLED';
+export const feedback = 'https://devsummit.io/feedback.html';
```
Comment (andy-shi88) [inline app/containers/Settings/constants.js:7]: it should be in the global constants

**Review by andy-shi88 — APPROVED** [review]

**Pull request merged**